castnet/__init__.py

```diff
@@ -452,6 +467,19 @@ def generic_patch(self, request):
             )
         try:
             cypher, params = self.request_to_cypher(label, resource_id, request.json)
+            cypher = (
```

Comment:
Maybe we should move this to the "request_to_cypher" function, or maybe a separate "add_history" function to avoid the duplicated code here and on 425. What do you think?